### ISSUE TITLE
[BUG for V68] map refuses to load in with LLL on 

### ISSUE THREAD

**Issue by Aono100** [IAmBatby/LethalLevelLoader#151]
Like the title says, i load up a normal map, and apparently LLL makes the game get stuck on the seeding screen, and the ship will be stuck and i can move around.  This happens 5 or 6 times in a row after a restart. and even after i complete a day. my only option was to uninstall it  so the game loads in right. i hope it get fixed soon 🙏

and to note, this mostly only happens on experimentation so far 

https://github.com/user-attachments/assets/9182e525-4b86-4041-bd41-12242b56c0ca

**Comment by WhoWhoa0** [comment]
Have you found a fix? Me and my friends have the same issue

**Comment by Aono100** [comment]
Nope, might be a code thing. tho, im no wizz at this type of stuff. Lol

**Comment by AdamABWozniak** [comment]
It also happens in my coop sessions (crew of 4), most often on March.
Here's code to my mods profile: 0193f582-ae0b-f9ef-2766-82312a92be6d

**Comment by NeoPositive** [comment]
It's due to LethalLib.

[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
LethalLevelLoader.DungeonLoader.PatchDynamicGlobalProps (DunGen.DungeonGenerator dungeonGenerator, LethalLevelLoader.ExtendedDungeonFlow extendedDungeonFlow) (at ./Loaders/DungeonLoader.cs:132)
LethalLevelLoader.DungeonLoader.PrepareDungeon () (at ./Loaders/DungeonLoader.cs:47)
(wrapper dynamic-method) LethalLevelLoader.Patches.DMD<LethalLevelLoader.Patches::DungeonGeneratorGenerate_Prefix>(DunGen.DungeonGenerator)
(wrapper dynamic-method) DunGen.DungeonGenerator.DMD<DunGen.DungeonGenerator::Generate>(DunGen.DungeonGenerator)
DunGen.RuntimeDungeon.Generate () (at <bdf6a080e98a49fd84b92b24894f768c>:IL_002C)
LethalLevelLoader.Patches.InjectHostDungeonSizeSelection (RoundManager roundManager) (at ./General/Patches.cs:643)
(wrapper dynamic-method) RoundManager.DMD<RoundManager::GenerateNewFloor>(RoundManager)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Trampoline<RoundManager::GenerateNewFloor>?1807613992(RoundManager)
LethalLevelLoader.LethalLibPatches.Dungeon_GenerateNewFloor_Prefix (System.Action`1[T] orig, RoundManager self) (at ./General/OptionalPatches/LethalLibPatches.cs:26)
(wrapper dynamic-method) LethalLib.Modules.Dungeon.DMD<LethalLib.Modules.Dungeon::RoundManager_GenerateNewFloor>(On.RoundManager/orig_GenerateNewFloor,RoundManager)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Hook<RoundManager::GenerateNewFloor>?1521231172(RoundManager)
LethalLevelLoader.LethalLevelLoaderNetworkManager.SetRandomExtendedDungeonFlowClientRpc (LethalLevelLoader.LethalLevelLoaderNetworkManager+StringContainer[] dungeonFlowNames, System.Int32[] rarities) (at ./Patches/LethalLevelLoaderNetworkManager.cs:138)
LethalLevelLoader.LethalLevelLoaderNetworkManager.__rpc_handler_2948417154 (Unity.Netcode.NetworkBehaviour target, Unity.Netcode.FastBufferReader reader, Unity.Netcode.__RpcParams rpcParams) (at <7d065480070b4244997d1d3dd40e6646>:IL_00B8)
Unity.Netcode.RpcMessageHelpers.Handle (Unity.Netcode.NetworkContext& context, Unity.Netcode.RpcMetadata& metadata, Unity.Netcode.FastBufferReader& payload, Unity.Netcode.__RpcParams& rpcParams) (at <895801699cfc4b4ab52267f31e2a4998>:IL_004E)
Rethrow as Exception: Unhandled RPC exception!
UnityEngine.Debug:LogException(Exception)
Unity.Netcode.RpcMessageHelpers:Handle(NetworkContext&, RpcMetadata&, FastBufferReader&, __RpcParams&)
Unity.Netcode.ClientRpcMessage:Handle(NetworkContext&)
Unity.Netcode.NetworkMessageManager:ReceiveMessage(FastBufferReader, NetworkContext&, NetworkMessageManager)
Unity.Netcode.NetworkMessageManager:HandleMessage(NetworkMessageHeader&, FastBufferReader, UInt64, Single, Int32)
Unity.Netcode.NetworkMessageManager:ProcessIncomingMessageQueue()
Unity.Netcode.NetworkManager:NetworkUpdate(NetworkUpdateStage)
Unity.Netcode.NetworkUpdateLoop:RunNetworkUpdateStage(NetworkUpdateStage)
Unity.Netcode.<>c:<CreateLoopSystem>b__0_0()

**Comment by NeoPositive** [comment]
It's due to LethalLib.

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
LethalLevelLoader.DungeonLoader.PatchDynamicGlobalProps (DunGen.DungeonGenerator dungeonGenerator, LethalLevelLoader.ExtendedDungeonFlow extendedDungeonFlow) (at ./Loaders/DungeonLoader.cs:132)
LethalLevelLoader.DungeonLoader.PrepareDungeon () (at ./Loaders/DungeonLoader.cs:47)
(wrapper dynamic-method) LethalLevelLoader.Patches.DMD<LethalLevelLoader.Patches::DungeonGeneratorGenerate_Prefix>(DunGen.DungeonGenerator)
(wrapper dynamic-method) DunGen.DungeonGenerator.DMD<DunGen.DungeonGenerator::Generate>(DunGen.DungeonGenerator)
DunGen.RuntimeDungeon.Generate () (at <bdf6a080e98a49fd84b92b24894f768c>:IL_002C)
LethalLevelLoader.Patches.InjectHostDungeonSizeSelection (RoundManager roundManager) (at ./General/Patches.cs:643)
(wrapper dynamic-method) RoundManager.DMD<RoundManager::GenerateNewFloor>(RoundManager)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Trampoline<RoundManager::GenerateNewFloor>?1807613992(RoundManager)
LethalLevelLoader.LethalLibPatches.Dungeon_GenerateNewFloor_Prefix (System.Action`1[T] orig, RoundManager self) (at ./General/OptionalPatches/LethalLibPatches.cs:26)
(wrapper dynamic-method) LethalLib.Modules.Dungeon.DMD<LethalLib.Modules.Dungeon::RoundManager_GenerateNewFloor>(On.RoundManager/orig_GenerateNewFloor,RoundManager)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Hook<RoundManager::GenerateNewFloor>?1521231172(RoundManager)
LethalLevelLoader.LethalLevelLoaderNetworkManager.SetRandomExtendedDungeonFlowClientRpc (LethalLevelLoader.LethalLevelLoaderNetworkManager+StringContainer[] dungeonFlowNames, System.Int32[] rarities) (at ./Patches/LethalLevelLoaderNetworkManager.cs:138)
LethalLevelLoader.LethalLevelLoaderNetworkManager.__rpc_handler_2948417154 (Unity.Netcode.NetworkBehaviour target, Unity.Netcode.FastBufferReader reader, Unity.Netcode.__RpcParams rpcParams) (at <7d065480070b4244997d1d3dd40e6646>:IL_00B8)
Unity.Netcode.RpcMessageHelpers.Handle (Unity.Netcode.NetworkContext& context, Unity.Netcode.RpcMetadata& metadata, Unity.Netcode.FastBufferReader& payload, Unity.Netcode.__RpcParams& rpcParams) (at <895801699cfc4b4ab52267f31e2a4998>:IL_004E)
Rethrow as Exception: Unhandled RPC exception!
UnityEngine.Debug:LogException(Exception)
Unity.Netcode.RpcMessageHelpers:Handle(NetworkContext&, RpcMetadata&, FastBufferReader&, __RpcParams&)
Unity.Netcode.ClientRpcMessage:Handle(NetworkContext&)
Unity.Netcode.NetworkMessageManager:ReceiveMessage(FastBufferReader, NetworkContext&, NetworkMessageManager)
Unity.Netcode.NetworkMessageManager:HandleMessage(NetworkMessageHeader&, FastBufferReader, UInt64, Single, Int32)
Unity.Netcode.NetworkMessageManager:ProcessIncomingMessageQueue()
Unity.Netcode.NetworkManager:NetworkUpdate(NetworkUpdateStage)
Unity.Netcode.NetworkUpdateLoop:RunNetworkUpdateStage(NetworkUpdateStage)
Unity.Netcode.<>c:<CreateLoopSystem>b__0_0()
```
[log.txt](https://github.com/user-attachments/files/18403132/log.txt)
